pages/version-policy/index.md

```diff
@@ -396,19 +396,19 @@ The designations are as follows:
         <td><p style="text-align: center;">⚫</p></td>
     </tr>
     <tr>
-        <td>Kafka-Zookeeper 2.4.x-3.4.13 (Recommended)</td>
+        <td>Kafka-Zookeeper 2.5.x-3.4.13 (Recommended)</td>
         <td><p style="text-align: center;">⚫</p></td>
         <td><p style="text-align: center;">⚫</p></td>
         <td><p style="text-align: center;">⚫</p></td>
     </tr>
     <tr>
-        <td>Kibana 2.4.x-5.6.5</td>
+        <td>Kibana 2.5.x-6.3.2</td>
```

Comment:
@vishnu2kmohan Could you share the link for Kibana documentation?

Reply:
There are no explicit docs for Kibana since it's a part of the Elastic release.
cc @samvantran @mpereira